docbook/Admin_Guide/en-US/Configuration.xml

```diff
@@ -5,6 +5,59 @@
 <chapter id="admin.config">
 	<title>Configuration</title>
 
+	<para>MantisBT is highly customizable through the web interface and
+		configuration files. Configuration options can be set globally as well as
+		customized for a specific project or user (except for options listed in
+		<emphasis>$g_global_settings</emphasis>).
+	</para>
+
+	<para>Configuration options can be set in
+		<emphasis>config_inc.php</emphasis> and in the
+		<emphasis>database</emphasis> (using the various manage pages).
+		Values stored in the database take precedence over values defined in
+		<emphasis>config_inc.php</emphasis>. Options stored in the database
+		can also be viewed and updated on
+		"Manage > Manage Configuration > Configuration Report" page.
+	</para>
+
+	<para>The sequence below describes how MantisBT determines which value
+		to use if an option was overridden:
+	</para>
+
+	<orderedlist>
+		<listitem>
+			<para>Look for the specified configuration option in the
+				database for current user and current project.
+			</para>
+		</listitem>
+		<listitem>
+			<para>Look for the specified configuration option in the
+				database for current user and all projects.
+			</para>
+		</listitem>
+		<listitem>
+			<para>Look for the specified configuration option in the
+				database for all users and current project.
+			</para>
+		</listitem>
+		<listitem>
+			<para>Look for the specified configuration option in the
+				database for all users and all projects.
+			</para>
+		</listitem>
+		<listitem>
+			<para>Look for the specified configuration option in
+				<emphasis>config_inc.php</emphasis> (where you might have some
```

Comment:
My post in the forum has been written for troubleshooting of this specific issue, not as some kind of documentation or howto.

I think it's better to understand if there is something similar to  what I wrote in the post:

> I recommend database configuration if you want to set options for a special project but not all projects, or a special user, but not all users.

Specific might be a better word than special for it.
I am no native English speaker, that's why I hesitate to write documentation ;-)